glossary.yml

```
@@ -6028,7 +6036,15 @@
 
 
 - slug: training_data
-
+  ref:
+    - machine_learning
+  en:
+    term: "training data"
+    def: >
+      Training data is a portion of a dataset used to train a
+      [machine learning](#machine_learning) algorithm to recognize similar data. It should always
+      be separated from the [test data](#test_data) to ensure that the model is 
+      properly tested with data it has never seen before.
```

```suggestion
      Training data is a portion of a dataset used to train [machine learning](#machine_learning)
      algorithm to recognise similar data. It should always be separated from the
      [test data](#test_data) to ensure that the model is properly tested with data it has
      never seen before.
```

Changing line widths and American to British spelling.